Laravel-Backpack/CRUD
# fix error display when using prefix in fields

## WHY

### BEFORE - What was wrong? What was happening before this PR?

Reported in https://github.com/Laravel-Backpack/CRUD/issues/4835

When adding `prefix => 'some_prefix'` in the field configuration, the errors are not properly displayed.

This was due to the fact that when there is a `prefix` in a field it wraps the input in a new div `input-group` to display the inline prefix with the field. 

### AFTER - What is happening after this PR?

The prefixed fields get the same css as the non prefixed and display the errors accordingly. 


## HOW

### How did you achieve that, in technical terms?

Replaced `.parent` with `.closest`, that way jQuery will search the parent and the parent parents until it finds the `form-group` element where the error classes should be added.
Also replaced `children` with `find`, because `children` will only search the direct childrens of the element and `find` will check deeper inside the childrens childrens. 

### Is it a breaking change?

I hope not, from my tests everything seems to be working fine after this PR.


### How can we test the before & after?

Add `prefix => '#'` in Monster `text` field (it's a required one). Trigger the validation error to see the errors display.
